lib/reporters/GruntCoverage.js

```diff
@@ -0,0 +1,64 @@
+define([
+    'dojo/node!istanbul/lib/collector',
+    'dojo/node!istanbul/lib/object-utils',
+    'dojo/node!grunt'
+], function (Collector, utils, grunt) {
+    function GruntCoverage(config) {
```

Comment:
Probably want to rename this to something like “CoverageEnforcementReporter” or something similar but better than what I can come up with :).